src/target/source/codegen_cuda.h

```diff
@@ -109,6 +109,13 @@ class CodeGenCUDA final : public CodeGenC {
   // Op attribute map
   OpAttrMap<bool> op_need_warp_shuffle_ = Op::GetAttrMap<bool>("cuda.need_warp_shuffle");
 
+  // The name of the barrier array in shared memory
+  const std::string barrier_name_ = "barrier";
+  // The size of the barrier array in shared memory
+  int barrier_count_ = -1;
+  // The alignment of the barrier array in shared memory
+  const int barrier_alignment_bytes_ = 64;
```

Comment:
Why is this 64 instead of 16?

Reply:
I chose 64 to match `kAllocAlignment` but the actual alignment requirement for bulk copy is 16.  It would work either way.  I  will use 16 and add a comment noting that it's the minimum.